src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -145,6 +146,14 @@ export default class ResultsViewPage extends React.Component<IResultsViewPagePro
             if (genes((window as any).serverVars.theQuery).length <= 1) {
                 $('a#mutex-result-tab').parent().hide();
             }
+
+            if (win.cancerStudyIdList !== 'null') {
```

Comment:
seems v tight with current JSP passing of variables? But fine for testing the new tracking